content/docs/introduction/getting_started.md

```diff
@@ -186,16 +169,13 @@ section in your `prometheus.yml` and restart your Prometheus instance:
 
 ```
 scrape_configs:
-  - job_name:       'example-random'
-
-    # Override the global default and scrape targets from this job every 5 seconds.
+  - job_name: 'example-random'
+    # Scrape targets from this job every 5 seconds.
     scrape_interval: 5s
-
     static_configs:
       - targets: ['localhost:8080', 'localhost:8081']
         labels:
           group: 'production'
```

Comment:
Adding a short comment that this will add group=production and canary as a label could be useful.